### PR TITLE
Revised Thai translations for v4

### PR DESCRIPTION
**Pull request by encX**
Part of #191

### CODE REVIEW

File: messages/index.js [hunk 1]
```diff
@@ -362,7 +362,7 @@ module.exports = {
     ru: "{{#def.n}}должно иметь не более, чем {{=n}} элемент{{#def.mPlural}}",
     sk: "{{#def.n}}nesmie obsahovať viac než {{=n}} prv{{#def.mPlural}}",
     sv: "{{#def.n}}borde inte ha fler än {{=n}} sak{{#def.mPlural}}",
-    th: "{{#def.n}}ควรมีสมาชิกไม่เกิน {{=n}}",
+    th: "{{#def.n}}ต้องมีสมาชิกไม่เกิน {{=n}}",
```
Comment (epoberezkin) [inline messages/index.js:365]: please double check - elements?

File: messages/index.js [hunk 2]
```diff
@@ -656,6 +657,7 @@ module.exports = {
     nl: "{{#def.n}}mag niet meer dan {{= n}} item{{#def.mPlural}} bevatten",
     "pt-BR": "{{#def.n}}não pode possuir mais que {{=n}} {{? n==1 }}item{{??}}itens{{?}}",
     ru: "{{#def.n}}должно иметь не более, чем {{=n}} элемент{{#def.mPlural}}",
+    th: "{{#def.n}}ต้องมีไม่เกิน {{=n}} ตัว",
```
Comment (epoberezkin) [inline messages/index.js:660]: please double check - elements?

File: messages/index.js [hunk 3]
```diff
@@ -140,7 +140,7 @@ module.exports = {
     ru: "{{#def.n}}должно иметь не более, чем {{=n}} элемент{{#def.mPlural}}",
     sk: "{{#def.n}}nemôže obsahovať viac, než {{=n}} prv{{#def.mPlural}}",
     sv: "{{#def.n}}borde ha fler än {{=n}} sak{{#def.mPlural}}",
-    th: "{{#def.n}}ควรมีสมาชิกไม่เกิน {{=n}}",
+    th: "{{#def.n}}ต้องมีสมาชิกไม่เกิน {{=n}} ตัว",
```
Comment (epoberezkin) [inline messages/index.js:143]: please double check - elements?

**Review by epoberezkin — APPROVED** [review]

**Comment by epoberezkin** [comment]
Thank you! Please check the comments - maybe it's all ok.

**Comment by encX** [comment]
@epoberezkin Hey, sorry for late reply. I've checked all of those you mentioned, they're all appropriate translated to their context.

**Comment by epoberezkin** [comment]
great - thank you!

**Pull request merged**